### DISCUSSION TITLE
No Power/Cadence readings on Peloton

### DISCUSSION THREAD

**Discussion by TimKingtonFC** [doudar/SmartSpin2k#569] [Q&A]
I'm using a PelotonBike (not +), and I'm not getting any power or cadence readings.  Zwift running on the Peloton tablet sees the SS2K, but the values are always zero.  I've also tried looking at them in the app on the shifter screen, and on the internal web troubleshooting page.
The SS2K is connected to the Peloton with the Y cable in tablet mode, and is running 24.8.16 firmware.

**Comment by doudar** [comment]
In tablet mode (switch up), you need to be in a Peloton workout or free ride (so the tablet requests power data from the bike). With the switch down, you disconnect the Y cable from the back of the Peloton tablet, and then the SS2k will request the power data from the bike. 

Try that, let us know!

**Reply by TimKingtonFC** [reply]
Same result - just seeing zeros. Should I have to reboot ss2k to switch modes? I tried that, in any case. When starting zwift, the resistance turned a little bit a couple of times, but not getting any numbers.

**Reply by doudar** [reply]
Nope, the switch is just a physical switch which disconnects the TX line on the serial port (Because you can only have one device talking on a traditional serial connection) when it's switched up, so no reboot required.

If your connections are correct, and the bike has power, I'd suspect the "y" cable, or another hardware issue.

 It also might be worth selecting "none" as the bluetooth power meter option on the SmartSpin2k config app, but that shouldn't cause a problem unless it's actually connecting to a power meter that's broadcasting 0 within range. 
 
 Possibly @eMadman has other ideas. The Peloton, pretty much "just works" with the switch down and the tablet not connected to the Y cable..

**Reply by eMadman** [reply]
At this point, I'd say let's see what shows up on the logs from headless mode.  I'm curious if the data format is different from what's expected. Or if the data is even hitting the device... That would indicate a wiring or PCB issue.  

Can you capture a log using the instructions here? https://github.com/doudar/SmartSpin2k/wiki/Viewing-logs-via-UDP
